roaring/roaring.go

```diff
@@ -706,6 +706,28 @@ func (b *Bitmap) Check() error {
 	return a
 }
 
+//Perform a logical negate of the bits in the range [start,end].
+func (b *Bitmap) Flip(start, end uint64) *Bitmap {
+	result := NewBitmap()
+	itr := b.Iterator()
+	itr.Seek(start)
+	v, eof := itr.Next()
+	for i := start; i < end; i++ {
```

Comment:
This logic doesn't handle the case where `start` is greater than existing bits in the bitmap.
I think you need to handle the case `if v < start` before the for loop.
you can see an example of this in test `TestBitmap_Flip_Array`: change `results := bm.Flip(0, 5)` to `results := bm.Flip(2, 5)` and you still get the same count on results.